### PR TITLE
feat-11-added the snackbar component and styles

### PR DESCRIPTION
**Pull request by SinchanaMS**
Created the Snackbar component.

**Files to Check:**
1. snackbars.html
2. snackbars.css

The other files can be ignored, only the nav links and documentation colors have been updated to reflect snackbars.

### **Preview:**

1. Simple Snackbars without Icons
image

2. Snackbars with Icons
image

### CODE REVIEW

File: docs/snackbars.html [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        integrity="sha512-IaaKO80nPNs5j+VLxd42eK/7sYuXQmr+fyywCNA0e+C6gtQnuCXNtORe9xR4LqGPz5U9VpH+ff41wKs/ZmC3iA=="
+        crossorigin="anonymous" referrerpolicy="no-referrer">
+    </script>
+    <script>
+        hljs.highlightAll();
+    </script>
+    <script src="/js/codeblock.js"></script>
+</body>
+
+</html>
```
Comment (deekshithmd) [inline docs/snackbars.html:232]: Looks good for me, you can proceed.

**Review by deekshithmd — APPROVED** [review]

File: stylesheets/documentation.css [hunk 2]
```diff
@@ -400,6 +400,36 @@ pre {
...
+    background: var(--SIMPLE-SNACKBARS-SECTION);
+    padding: 1rem;
+    border-radius: 1rem;
+}
+
+.snackbars-simple {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0px;
```
Comment (rahulyadav139) [inline stylesheets/documentation.css:414]: I think the gap is by default set zero, you don't need to mention it.

**Review by rahulyadav139 — APPROVED** [review]

File: stylesheets/documentation.css (hunk 2 above)
Comment (SinchanaMS) [reply stylesheets/documentation.css:414]: Yes, it is by default 0. However, the component showcase section of my documentation, in general, has a gap that is being overridden here.

**Comment by SinchanaMS** [comment]
Thank you for the reviews! Merging to temp-dev.

**Pull request merged**